lisa/tools/gdb.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+
+class Gdb(Tool):
+    @property
+    def command(self) -> str:
+        return "gdb"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def debug(self, filename: str, arguments: str = "-batch -ex 'run' -ex 'bt'") -> str:
```

Comment:
The `arguments` is not good to be default, because it doesn't like reusable or common. So just make arguments required.

Reply:
updated